### ISSUE TITLE
[Hold for payment 2023-10-13] [ECard Settings] - Add “Reveal details” for the digital card

### ISSUE THREAD

**Issue by grgia** [Expensify/App#22874]
HOLD ON https://github.com/Expensify/App/issues/22873

## Design Doc Section
https://docs.google.com/document/d/1rFxJ78vz5On6zSWzYa51B9v-tyLdC5pUsBeLOLig0t4/edit#bookmark=id.gpdwt3bqorke
<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0191d13ceb3e2b2971</li>
        <li>Upwork Job ID: 1679790549101445120</li>
        <li>Last Price Increase: 2023-07-14</li>
    </ul>
</details>

**Comment by grgia** [comment]
cc @pac-guerreiro

**Comment by pac-guerreiro** [comment]
Could you assign me to this @grgia ? 😄

**Comment by marcaaron** [comment]
This one is held on the Expensify Card page I think because you need an actual virtual card. Though we could also start building out a component + story for this with test data. 

And then once the card page is ready plug it all in and hook up the API request to fetch and display the card details.

**Comment by JKobrynski** [comment]
Hi, I'm Julian from Callstack - expert contributor group - and I would like to take a look at this issue.

**Comment by JKobrynski** [comment]
Melvin, no overdue here, this issue is waiting for another issue to be merged.

**Comment by JKobrynski** [comment]
Melvin, no overdue here, this issue is waiting for another issue to be merged.

**Comment by grgia** [comment]
not overdue

**Comment by grgia** [comment]
We need to handle $500 C+ review payment to @eVoloshchak before closing @joekaufmanexpensify

**Comment by joekaufmanexpensify** [comment]
Cool cool, I can take this.

**Comment by joekaufmanexpensify** [comment]
@JKobrynski is with an agency, so their payment is handled separately. The only payment needed here is a $500 payment to @eVoloshchak for their C+ review of this PR. This will be paid via NewDot.

**Comment by joekaufmanexpensify** [comment]
@eVoloshchak could you please submit a request for $500 and let us know here once you've done that? TY!

**Comment by joekaufmanexpensify** [comment]
Oh yeah, good call. I will add a header to remember, and then follow up in 3 more days!

**Comment by joekaufmanexpensify** [comment]
@eVoloshchak mind requesting payment today? It's now been 7 days.

**Comment by eVoloshchak** [comment]
Requested the payment, apologies for the delay

**Comment by joekaufmanexpensify** [comment]
Great, thanks for confirming! Closing as payment has been requested. Payment summary message is [here. ](https://github.com/Expensify/App/issues/22874#issuecomment-1755428414)

**Comment by JmillsExpensify** [comment]
$500 payment approved for @eVoloshchak based on summary.
